src/index.js

```diff
@@ -0,0 +1,157 @@
+const city = document.getElementById('city_name')
+let inputCity = document.getElementById('city_input')
```

Comment:
```suggestion
const inputCity = document.getElementById('city_input')
```
Note these type of variables storing html elements can be declared as `const` variables since under the hood they're just objects, so you'll still be able to modify the data inside the object. This just ensures the variable itself can't be reassigned.